server/db/models.py

```diff
@@ -147,6 +149,25 @@
     Column('is_active',     Boolean,    nullable=False)
 )
 
+coop_leaderboard = Table(
```

Comment:
Please place tables in alphabetical order

Reply:
Ah, that makes sense :P I didn't notice and just tried to group them roughly by "concern" and then add new ones at the end.